src/primitives/transaction.h

```diff
@@ -113,13 +103,12 @@ class CPourTx
         return (
             a.vpub_old == b.vpub_old &&
             a.vpub_new == b.vpub_new &&
-            a.scriptPubKey == b.scriptPubKey &&
-            a.scriptSig == b.scriptSig &&
             a.anchor == b.anchor &&
             a.serials == b.serials &&
             a.commitments == b.commitments &&
             a.ciphertexts == b.ciphertexts &&
```

Comment:
Also swap commitments and ciphertexts here for consistency, even though it doesn't matter logically.